openshift/templates/issuer-agent/issuer-agent-deploy.prime-sandbox.dev.param

```diff
@@ -3,20 +3,21 @@
 # Component: .
 # Template File: ../openshift/templates/issuer-agent/issuer-agent-deploy.yaml
 #=========================================================
-# NAME=agent
-# APP_NAME=issuer-prime
+# NAME=agent-sandbox
+# APP_NAME=issuer
 # APP_GROUP=prime
 # SUFFIX=
```

Comment:
You're missing the suffix here, the deployment won't be grouped with the rest of the `prime` apps.